trade-finance-manager-ui/server/constants/teamIds.js

```diff
@@ -1,9 +1,17 @@
+/* eslint-disable max-len */
+/**
+ * @typedef {'UNDERWRITING_SUPPORT' | 'UNDERWRITER_MANAGERS' | 'UNDERWRITERS' | 'RISK_MANAGERS' | 'BUSINESS_SUPPORT' | 'PIM' | 'PDC_READ' | 'PDC_RECONCILE'} TEAM_IDS
+ */
+/* eslint-enable max-len */
```

Comment:
Might be worth changing this to a typescript declaration file

Reply:
Could I get someone else's opinion on this?

Reply:
Sounds good to me 👍 

I would also rename `TEAM_IDS` to `TeamIds` as this represents a type rather than an enum.